src/main/scripts/test-in-container

```diff
@@ -9,5 +9,5 @@ echo 'Testing tutorial in user environment...'
 make -j`nproc`
 cd ~
 echo "module TEST imports BOOL endmodule" > test.k
-kompile test.k --backend llvm
+kompile test.k --backend ocaml
```

Comment:
What's going on here? Why are we only running this smoke test with ocaml/haskell now?

Reply:
So, the smoke test exists in order to test that the backends that were not part of the K tutorial still work after being installed via packaging. We also run the tests in the K tutorial. In the past, the only backends not tested by the tutorial were the llvm and haskell backend. Now, they are the ocaml and haskell backends. Hence the change.